jobs/cluster/smoke-test/Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env groovy
+
+// A Job to run a smoke test in a cluster
+// 
+// It invokes the 'smoketest' operation of 'cicd-control.sh' for the specified cluster.
+// Optionally emails test results.
+
+node('openshift-build-1') {
+
+    properties([
+            [   $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        $class: 'hudson.model.StringParameterDefinition',
+                        name: 'CLUSTER_NAME',
+                        description: 'Cluster to run tests on',
+                    ],
+                    [
+                        $class: 'hudson.model.BooleanParameterDefinition',
+                        defaultValue: false,
+                        description: 'Send email notifications',
+                        name: 'MAIL_RESULTS'
```

Comment:
So: this disables email notifications by default (i.e. no emails would be sent on success or failure). Should we have a default of `true` here? I set it to `false` because the nature of the job seems to me it might not be worth sending notifications for a single smoke test run...

Moreover: the upgrade job calls this one with email notifications explicitly disabled too, because it handles its own email notifications.

Reply:
Seems like a reasonable way to start.